recipes/sentry-native/all/conanfile.py

```diff
@@ -73,6 +75,7 @@ def _configure_cmake(self):
         self._cmake.definitions["SENTRY_ENABLE_INSTALL"] = True
         self._cmake.definitions["SENTRY_TRANSPORT"] = self.options.transport
         self._cmake.definitions["SENTRY_PIC"] = self.options.get_safe("fPIC", True)
+        self._cmake.definitions["SENTRY_INTEGRATION_QT"] = self.options.get_safe("qt", False)
```

Comment:
```suggestion
        self._cmake.definitions["SENTRY_INTEGRATION_QT"] = self.options.qt
```
no need for get_safe if the option is never deleted